api/src/main/scala/io/phdata/rest/WorkspaceController.scala

```diff
@@ -198,7 +199,12 @@ class WorkspaceController[F[_]: Sync: Timer: ContextShift: ConcurrentEffect](
               case e: Throwable =>
                 logger.error(s"Failed to add member to workspace $id: ${e.getLocalizedMessage}", e).pure[F]
             }
-            _ <- emailService.newMemberEmail(id, memberRequest).value
+            emailResult = try {
+              emailService.newMemberEmail(id, memberRequest).value
```

Comment:
I know it's not part of this PR but maybe we should rename `newMemberEmail` to `sendNewMemberEmail` because it actually send an email.